File: 프로그래머스/0/181939. 더 크게 합치기/더 크게 합치기.java
class Solution {
    public int solution(int a, int b) {
        int answer = 0;
        int a_b = 0;
        int b_a = 0;
        a_b = Integer.parseInt("" + a + b);
        b_a = Integer.parseInt("" + b + a);
        if (a_b < b_a) {
            answer = b_a;
        } else {
            answer = a_b;
        }
        return answer;
    }
}

File: 프로그래머스/0/181939. 더 크게 합치기/더 크게 합치기.py
def solution(a, b):
    answer = 0
    # if int(str(a) + str(b)) < int(str(b) + str(a)):
    #     answer = int(str(b) + str(a))
    # else:
    #     answer = int(str(a) + str(b))
    answer = int(max(f"{a}{b}", f"{b}{a}"))
    return answer